ui/js/components/perf/compare.js

```diff
@@ -108,3 +108,52 @@ treeherder.component('compareError', {
         newRevision: '<'
     }
 });
+
+treeherder.component('distributionGraph', {
+    template: '<canvas id="a" width="400" height="30"></canvas>',
+    bindings: {
+        replicates: '<'
+    },
+    controller: [function () {
+        var ctrl = this;
+        var cvs = document.getElementById("a");
+        var ctx = cvs.getContext("2d");
+        var max = Math.ceil(Math.max.apply(null, ctrl.replicates)*1.01);
+        var min = Math.floor(Math.min.apply(null, ctrl.replicates)/1.01);
```

Comment:
Let's call these last two `maxValue` / `minValue`

Is there any reason you're using apply here rather than just `Math.max(ctrl.replicates)`?